charts/stable/appdaemon/questions.yaml

```diff
@@ -51,15 +51,15 @@ questions:
           schema:
             type: string
             default: ""
-            valid_chars: '^[0-9]{1,}\.{1}[0-9]{1,}$'
+            valid_chars: '^(\+|-)?(?:90(?:(?:\.0{1,15})?)|(?:[0-9]|[1-8][0-9])(?:(?:\.[0-9]{1,15})?))$'
             required: true
         - variable: LONGITUDE
           label: Longitude
           description: Longitude of your home assistant instance
           schema:
             type: string
             default: ""
-            valid_chars: '^[0-9]{1,}\.{1}[0-9]{1,}$'
+            valid_chars: '^(\+|-)?(?:180(?:(?:\.0{1,15})?)|(?:[0-9]|[1-9][0-9]|1[0-7][0-9])(?:(?:\.[0-9]{1,15})?))$'
```

Comment:
There is a validation.md file in docs, update it as well.
Doesn't it seems a bit too bit complex?
What extra validation does this adds?

Reply:
well...the original didnt accept negative longitudes, this one does and accepts upto 15 chars if needed

Reply:
Seems quite complex for just increasing to 15 chars + a symbol

What else does it match?

Reply:
well it actually detects legit long and lat values.